### PR TITLE
Documentation on setting up redis

### PR DESCRIPTION
**Pull request by neilisfragile**
https://github.com/matrix-org/synapse/issues/7402

### CODE REVIEW

File: docs/workers.md [hunk 1]
```diff
@@ -101,7 +121,8 @@ recommend the use of `systemd` where available: for information on setting up
 `systemd` to start synapse workers, see
 [systemd-with-workers](systemd-with-workers). To use `synctl`, see below.
 
-### Using synctl
```
Comment (neilisfragile) [inline docs/workers.md:104]: Should references to synctl be removed?

**Comment by neilisfragile** [comment]
TODO 
Paragraph starting "In the config file for each worker..." Needs Redis specific details

File: docs/workers.md (hunk 1 above)
Comment (richvdh) [reply docs/workers.md:104]: maybe, but not today imho

File: docs/workers.md [hunk 2]
```diff
@@ -27,50 +35,43 @@ the correct worker, or to the main synapse instance. Note that this includes
...
 replication listener port and `worker_replication_http_port` should point to
 the HTTP replication port.
 
-Currently, the `event_creator` and `federation_reader` workers require specifying
-`worker_replication_http_port`.
-
-For instance:
+For example:
 
     worker_app: synapse.app.synchrotron
```
Comment (clokep) [inline docs/workers.md:76]: Should we wrap this in yaml syntax highlighting while we're at it?

**Comment by clokep** [comment]
I gave this a read and it seems to make sense to me. 👍

**Pull request merged**